pkg/apis/istio/v1alpha1/istioinstaller_types.proto

```diff
@@ -0,0 +1,441 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+import "k8s.io/api/core/v1/generated.proto";
+import "k8s.io/api/autoscaling/v1/generated.proto";
+import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
+import "github.com/gogo/protobuf/protobuf/google/protobuf/wrappers.proto";
+
+message IstioInstaller {
+    InstallerSpec spec = 3;
+    InstallerStatus status = 4;
+}
+
+// InstallerSpec defines the desired state of IstioInstaller.
+message InstallerSpec {
+    // Prefix added to all namespaces for any installed component.
+    string default_namespace_prefix = 5;
+
+    // Selection and configuration of core Istio features.
+    TrafficManagementFeatureSpec traffic_management = 25;
+    PolicyTelemetryFeatureSpec policy_telemetry = 26;
```

Comment:
mentioned in the design doc... i prefer to separate the two given it is totally two different features for our users.

Reply:
I'm ok with splitting them up but I'd like to hear from @smawson as this is something that came out of early reviews.

Reply:
Checked with Sven, he's ok with it, done.